style.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:wght@300;400;600;700;900&family=Roboto:wght@500;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* the header section */
+header {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  padding: 25px;
+  width: 375px;
+  margin: auto;
+}
+
+#logo {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+}
+
+/* headline section */
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 24px 145px;
+  background-image: url(./Images/Group.png);
```

Comment:
- [ ] You will notice that this image came out well in the figma design, kindly adapt it to yours also.

## Figma
![image](https://user-images.githubusercontent.com/86200356/234810946-38842524-d82c-45be-bedb-698a9b23965c.png)

## Yours
![image](https://user-images.githubusercontent.com/86200356/234811061-5b889596-b9f3-49c5-a174-6da9940e6469.png)